packages/vscode-webui/src/components/worktree-select.tsx

```diff
@@ -33,7 +33,7 @@ const getWorktreeName = (worktree: GitWorktree | undefined) => {
     return;
   }
   if (worktree.isMain) {
-    return worktree.branch || "main";
```

Comment:
@zhanba fyi - we no longer display branch name in worktree select. just display workspace instead.